Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Card.css';
+
+function CardBanner(){
+    return(
+        <div className="bannerImg"></div>
```

Comment:
You _could_ do inline styles here so that it's clear what this is without going into your CSS:
```javascript
const bannerStyle = {
  'background-image': url("https://ibin.co/3wnC6SgIOJud.png"),
  'background-repeat': no-repeat,
  'background-size': cover,
  'background-position': center,
  height: 25rem
};
return <div style={bannerStyle}></div>;
```